torch/onnx/symbolic_opset13.py

```diff
@@ -109,6 +109,136 @@ def unsafe_split_with_sizes(g, self, split_sizes, dim, _outputs=None):
     return split_with_sizes(g, self, split_sizes, dim, _outputs)
 
 
+@symbolic_helper.parse_args("v", "v", "i", "i")
+def tensor_split(g, self, indices_or_sections, dim, _outputs=None):
+    axis = g.op("Constant", value_t=torch.tensor(dim, dtype=torch.long))
+    axis = opset11.unsqueeze(g, axis, 0)
+
+    def static_tensor_split(self, indices):
```

Comment:
since this is only used in one place, just inline the code rather than defining a helper function